### PR TITLE
v3 ✨

### PR DESCRIPTION
**Pull request by keirog**
Little bit of a tidy up for our foundations.

Deals with these items: https://github.com/Financial-Times/n-ui-foundations/issues/34

### CODE REVIEW

File: main.scss [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
-This includes all the basic origami styles and mixins you can
+This includes all the basic Origami styles and mixins you can
 expect to be available when a page uses n-ui
 
 */
 
 // Base unit to standardize spacing
 // Decide where it's gonna live later
-$spacing-unit: 16px;
+$spacing-unit: 20px;
```
Comment (wheresrhys) [inline main.scss]: this used much?

File: util/main.scss [hunk 2]
```diff
@@ -30,10 +27,6 @@
 		}
 	}
 
-	.n-util-visually-hidden {
-		@include nUtilsVisuallyHidden;
-	}
-
 	.no-js .n-util-hide-no-js,
 	.js .n-util-hide-js,
 	.core .n-util-hide-core,
```
Comment (wheresrhys) [inline util/main.scss]: would getting rid of the n-util namespace entirely be possible, so these go to n-ui-hide-no-js etc

Comment (keirog) [reply util/main.scss]: oh yeah, good shout. I'll update and add it to the migration guide

File: main.scss (hunk 1 above)
Comment (keirog) [reply main.scss]: only [here and there](https://github.com/search?q=org%3AFinancial-Times+spacing-unit&type=Code), but the plan was to align it with what the design folks want (& use), and leave any decisions about its adoption/usefulness for another time. Seemed like a bit of a rabbit hole

**Pull request merged**